src/Compilers/CSharp/Test/Symbol/DocumentationComments/ParameterTests.cs

```diff
@@ -833,6 +832,23 @@ class C<T>
             Assert.Equal(expectedValueParameter, model.GetSymbolInfo(names[11]).Symbol);
         }
 
+        [Fact]
+        public void ExtensionMethodsAreNotAvailableInEarlierCSharpVersions()
+        {
+            var code = @"
+ public static class Test
+ {
+     public static void DoSomething(this int x) { }
+ }";
+
+            CreateCompilation(code, parseOptions: new CSharpParseOptions(LanguageVersion.CSharp2)).VerifyDiagnostics(
```

Comment:
`TestOptions.Regular.WithLanguageVersion(LanguageVersion.CSharp2)`

Reply:
This is a test move. Did you want the setup changed. Generally I thought that was a no no.